sass/_CategoryFacet.scss

```diff
@@ -0,0 +1,86 @@
+@import 'Variables';
+
+.CoveoCategoryFacet {
+  transition: height 0.4s $standard-transition-timing-function;
+  &.coveo-hidden {
+    display: none;
+  }
+
+  & > .coveo-facet-values > .coveo-category-facet-value {
```

Comment:
There's a lot of very precise selectors in general in this file.

Any way we can make those less specific when possible ? Much easier to customize.

Reply:
I used the less precise selectors that I could. Because of the way the HTML is built, I need to use the direct child selector to select a specific category value.